web/src/pages/news/blogs/[slug].page.test.tsx

```diff
@@ -0,0 +1,393 @@
+import { render, screen } from "@testing-library/react";
+import { GetServerSidePropsContext } from "next";
+
+import { BlogPostStoryblok } from "@/types/storyblok";
+import {
+	getSlugFromParams,
+	fetchStory,
+	getStoryVersionFromQuery,
+} from "@/utils/storyblok"; // replace with actual path
+
+import BlogPostPage, { getServerSideProps } from "./[slug].page";
+
+jest.mock("@/utils/storyblok", () => jest.fn()); // replace with actual path
+
+const mockArticle = {
```

Comment:
This is definitely a candidate to swap out for the new utility objects, this thing is massive! Good effort doing it in the first place though